package.json

```diff
@@ -38,9 +38,9 @@
   "license": "MIT",
   "peerDependencies": {
     "@capacitor/core": ">=3.0.0",
-    "@sentry/angular": "7.56.0",
-    "@sentry/react": "7.56.0",
-    "@sentry/vue": "7.56.0"
+    "@sentry/angular": "^7.56.0",
+    "@sentry/react": "^7.56.0",
+    "@sentry/vue": "^7.56.0"
```

Comment:
I am aware that it may be annoying, but we planned to only use fixed versions, so instead, I would recommend changing those versions to "7.60.1" instead of "^7.56.0"

This may be mitigated in the future if we do more frequent releases bumping Sentry JavaScript

Reply:
But you don't follow the sentry official version when they are published

Reply:
@lucas-zimerman what's the reason for this?

Reply:
The main reason is we want to make sure that the release of capacitor SDK works with the given Sentry Siblings releases.

Are you not able to install Sentry Capacitor version 0.12.2 with Sentry React 7.56.0?

Reply:
Okay, but in the semantic versioning the second digit is the minor version, if you don't need the features of the 7.62.x you can stay in 7.56.0 in your minimal dependencies and you still stay compatible with the nexts minor versions.